### DISCUSSION TITLE
Authentication & need of user.read permissions

### DISCUSSION THREAD

**Discussion by arajawat** [microsoft/Microsoft-Fabric-workload-development-sample#166] [Q&A] [answered]
Following the [authentication tutorial](https://learn.microsoft.com/en-us/fabric/workload-development-kit/authentication-tutorial#add-api-permissions), the image in doc talks about adding certain set of scopes for sample workload to run.

I want to know, what is the need of "User.read" scope from Graph APIs. I have not provided permission to this scope from security purpose and only enabled my Azure App to be authorized on required scopes like "Lakehouse.Read.All". Am I missing something under the hood and is this user scope a must, to have our workload running? What's the use of this scope from workload perspective?

![image](https://github.com/user-attachments/assets/4ef02175-4e42-4e40-abbb-ab8745742df7)

**Comment by ramizsha** [comment, marked as answer]
This scope is added automatically when creating the application, may I ask what's the security concern here?.
Our consent flow requires at least 1 dependency on graph and it's recommended to keep it.
You can replace it with any other graph scope including [openID Connect scopes](https://learn.microsoft.com/entra/identity-platform/scopes-oidc#openid-connect-scopes), for example "openId".
